src/standard.ts

```diff
@@ -35,10 +41,14 @@ export class StandardMerkleTree<T extends any[]> {
       ]));
   }
 
-  static of<T extends any[]>(values: T[], leafEncoding: string[]) {
-    const hashedValues = values
-      .map((value, valueIndex) => ({ value, valueIndex, hash: standardLeafHash(value, leafEncoding) }))
-      .sort((a, b) => compareBytes(a.hash, b.hash));
+  static of<T extends any[]>(values: T[], leafEncoding: string[], options: StandardMerkleTreeOptions = {}) {
+    const { sortLeaves } = { ...defaultOptions, ...options };
```

Comment:
The user can pass `options = { sortLeaves: undefined }`, and this code would not fallback to the default.

This fixes that:

```suggestion
    const sortLeaves = options.sortLeaves ?? defaultOptions.sortLeaves;
```